activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -373,6 +380,17 @@ def group!(*args) # :nodoc:
     #
     #   User.order('name DESC, email')
     #   # SELECT "users".* FROM "users" ORDER BY name DESC, email
+    #
+    # This could be a source of SQL injection, so only strings composed of plain column names and simple function(column_name) expressions with optional ASC/DESC modifiers are allowed.
```

Comment:
We try to keep line lengths around 80 characters for new code:
```suggestion
    # This could be a source of SQL injection, so only strings composed of plain column names
    # and simple function(column_name) expressions with optional ASC/DESC modifiers are allowed.
```